cmd/ethermintd/root.go

```diff
@@ -114,6 +114,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		rpc.StatusCommand(),
 		queryCommand(),
 		txCommand(),
+		ledgerCommand(),
```

Comment:
we should remove this and just fix the `keys add --ledger` command

Reply:
i  inspected cosmos level modification, that requires overriding whole cosmos-sdk.
but cosmo-sdk ledger is not possible, because eth ledger app only working with eth style signing.

Reply:
another solution is using

https://github.com/LedgerHQ/ledgerjs/tree/master/packages/hw-app-eth#signeip712hashedmessage

just signing with hash?
then we can do it in cosmos-level.

- good: supporting all
- bad: signer cannot see contents of the message to sign

Reply:
i'll check signing hash